src/components/survey/Survey.tsx

```diff
@@ -1,39 +1,17 @@
 import React from 'react';
-import { QuestionView, Question } from '../question/Question';
+import { QuestionGroupView, Question } from '../question/Question';
 
-export default class ExamGeneratorView extends React.Component {
+export default function ExamGeneratorView() {
 
-  questionViews: JSX.Element[] = [];
-  questionViewRefs: React.RefObject<QuestionView>[] = [];
-  state = {
-    questionViews: this.questionViews
-  }
-
-  addNewQuestion = () => {
-    this.questionViewRefs.push(React.createRef<QuestionView>());
-    this.questionViews.push(
-      <QuestionView
-        ref={this.questionViewRefs[this.questionViewRefs.length - 1]}
-        id={this.questionViewRefs.length} />);
-    this.setState({ questions: this.questionViews });
-  }
-
-  createExam = () => {
-    const questions: Question[] = [];
-    this.questionViewRefs.forEach(ref => {
-      const question: Question | undefined = ref.current?.getQuestion();
-      if (question !== undefined)
-        questions.push(question);
-    });
+  const questions: Question[] = [];
 
+  const finishSurvey = () => {
     console.log(JSON.stringify(questions));
-  }
+  };
 
-  render() {
-    return (<div className="generatorView">
-      {this.questionViews.map(question => question)}
-      <button onClick={this.addNewQuestion}>Add Question</button>
-      <button onClick={this.createExam}>Finish Survey</button>
-    </div>);
-  }
-}
\ No newline at end of file
+  return (<div>
+    <QuestionGroupView questionList={questions}></QuestionGroupView>
```

Comment:
We could use just QuestionGroup instead of QuestionGroupView for naming convention.